website/style-dark.css

```diff
@@ -36,18 +36,22 @@ a.navbar-brand:hover {
 
 .container-logo {
   display: flex;
+  margin: 10px;
 }
 .imglogo {
-  width: 8%;
+  width: 20%;
```

Comment:
Please don't change anything in the logo. We only need to change the heading text.

Reply:
the change has been done.